src/styles/blocks/header.scss

```diff
@@ -0,0 +1,62 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: $header-height;
+  padding-right: 50px;
+  font-weight: 500;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  text-decoration: none;
+  margin-left: 50px;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+  }
+
+  &__item {
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    display: inline-block;
+    line-height: $header-height;
+    color: $nav-link-color;
+    text-decoration: none;
+    text-transform: uppercase;
+    position: relative;
+
+    transition-duration: $anim-duration;
+
+    &:hover {
+      color: $secondary-blue-accent-color;
+    }
+
+    &.is-active {
```

Comment:
better don't insert one class inside the other one
 
<img width="1018" alt="Screenshot 2024-03-13 at 16 19 17" src="https://github.com/mate-academy/layout_catalog/assets/45732323/365fef8e-fadd-4f04-931d-c9be76d2adf1">